src/lib/deploy.ml

```diff
@@ -1109,6 +1136,65 @@ module Cluster = struct
     in
     return conf
 
+  let biokepi_machine
+      ?(gatk_jar_location = `Fail "Cannot use GATK without the JAR location")
+      ?(mutect_jar_location = `Fail "Cannot use Mutect without the JAR location")
+      t ~configuration ~work_dir =
+    let host = ketrew_host t ~work_dir in
+    let max_processors =
+      match t.torque_server.Node.machine_type with
+      | `Google_cloud g ->
+        begin match g with
+        | `Small -> 1
+        | `Highmem_8 -> 8
```

Comment:
8 if the number of processors. 
The specs are: “`n1-highmem-8 (8 vCPUs, 52 GB memory)`”
I use these for now, I plan to run more benchmarking to see what we really need.
Also for now the PBS cluster is configured to be homogeneous; it has some "self-config" capabilities; but I don't know yet how to do a proper heterogeneous cluster.